helm-chart/sefaria-project/templates/configmap/local-settings-file.yaml

```diff
@@ -179,11 +179,10 @@ data:
 
     # Turns off search autocomplete suggestions, which are reinitialized on every server reload
     # which can be annoying for local development.
-    DISABLE_AUTOCOMPLETER = False
-    ENABLE_LINKER = False
+    DISABLE_AUTOCOMPLETER = os.getenv("DISABLE_AUTOCOMPLETER", "false") == "true"
+    ENABLE_LINKER = os.getenv("ENABLE_LINKER", "false").lower == "true"
```

Comment:
@BrendanGalloway I think you meant `.lower()` with parentheses. Also, should you use `.lower()` on the line above as well?